apps/user-profile/src/components/modules/OnboardingLandingPage/OnboardingLandingPage.js

```diff
@@ -1,272 +1,385 @@
-    ProfileContainer, ProfileHeader
-} from './StyledOnboardingLandingPage'
+  ButtonContainer,
+  OnboardingContainer,
+  FormContainer,
+  PageContainer,
+  BannerContainer,
+  FormFields,
+  FormFooter,
+  ProfileContainer,
+  UploadButton,
+  ProfileHeader,
+  ProfilePicture,
+  UploadedProfilePicture,
+} from './StyledOnboardingLandingPage';
 
 const initialFormValue = {
-    firstName: '',
-    firstNameTouched: false,
-    lastName: '',
-    lastNameTouched: false,
-    location: '',
-    role: '',
-    headline: '',
-    termsAndConditions: false,
-    emailMarketing: false
-}
+  firstName: '',
+  firstNameTouched: false,
+  lastName: '',
+  lastNameTouched: false,
+  location: '',
+  role: '',
+  headline: '',
+  termsAndConditions: false,
+  emailMarketing: false,
+};
 
 const initialFormValidation = {
-    isFirstNameValid: /^[a-z ,.'-]+$/i.test(initialFormValue.firstName),
-    isLastNameValid: /^[a-z ,.'-]+$/i.test(initialFormValue.lastName)
-}
+  isFirstNameValid: /^[a-z ,.'-]+$/i.test(initialFormValue.firstName),
+  isLastNameValid: /^[a-z ,.'-]+$/i.test(initialFormValue.lastName),
+};
 
 export default function OnboardingLandingPage() {
-    const [person, setPerson] = useState(initialFormValue);
-    const [formValidation, setFormValidation] = useState(validate(initialFormValue));
-    const router = useRouter()
-    let isFormValid = false;
-    useEffect(() => {
-        setFormValidation(validate(person));
-    }, [person]);
+  const { userData } = useUserDataContext();
+  const [person, setPerson] = useState({
+    ...initialFormValue,
+  });
+  const [formValidation, setFormValidation] = useState(
+    validate(initialFormValue)
+  );
+  const [profilePicture, setProfilePicture] = useState(
+    userData?.profile?.profilePictureUrl
+  );
+  const [uploadedProfilePicture, setUploadedProfilePicture] = useState(null);
+
+  const router = useRouter();
+  let isFormValid = false;
+  useEffect(() => {
+    setFormValidation(validate(person));
+  }, [person]);
+
+  function validate(formValue) {
+    return {
+      isFirstNameValid:
+        formValue.firstNameTouched &&
+        /^[a-z ,.'-]+$/i.test(formValue.firstName),
+      isLastNameValid:
+        formValue.lastNameTouched && /^[a-z ,.'-]+$/i.test(formValue.lastName),
+      isFormValid: function () {
+        return (
+          formValue.termsAndConditions &&
+          this.isFirstNameValid &&
+          this.isLastNameValid
+        );
+      },
+    };
+  }
 
-    function validate(formValue) {
-        return {
-            isFirstNameValid: formValue.firstNameTouched && /^[a-z ,.'-]+$/i.test(formValue.firstName),
-            isLastNameValid: formValue.lastNameTouched && /^[a-z ,.'-]+$/i.test(formValue.lastName),
-            isFormValid: function () { return formValue.termsAndConditions && this.isFirstNameValid && this.isLastNameValid }
+  function handleFileChange(event) {
+    const file = event.target.files[0];
+    if (file) {
```

Comment:
Could we combine this and the next if check using `if (file?.type === 'image/png' || file?.type === 'image/jpeg') `?